app/controllers/mixins/checked_id_mixin.rb

```diff
@@ -104,4 +104,36 @@ def find_record_with_rbac_flash(model, id, resource_name = nil)
 
     checked_object
   end
+
```

Comment:
Fixed in https://github.com/ManageIQ/manageiq-ui-classic/pull/935

Thanks, @martinpovolny